Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN python -m spacy download en_core_web_sm
 
 WORKDIR /app
 RUN chown -R $USER:$USER /app/
```

Comment:
This `RUN chown` could be removed if you put `USER` instruction before `WORKDIR`.